functions/assignOfflineContact.ts

```diff
@@ -61,33 +73,78 @@ const assignToOfflineWorker = async (
 ) => {
   const previousActivity = targetWorker.activitySid;
   const previousAttributes = JSON.parse(targetWorker.attributes);
+
   const availableActivity = await context
     .getTwilioClient()
     .taskrouter.workspaces(context.TWILIO_WORKSPACE_SID)
     .activities.list({ friendlyName: 'Available' });
 
   await targetWorker.update({
     activitySid: availableActivity[0].sid,
-    attributes: JSON.stringify({ ...previousAttributes, waitingOfflineContact: true, acceptOnlyTask: newTask.sid }), // waitingOfflineContact & acceptOnlyTask are routing rules used to avoid other tasks to be assigned during this window of time
+    attributes: JSON.stringify({ ...previousAttributes, waitingOfflineContact: true }), // waitingOfflineContact is used to avoid other tasks to be assigned during this window of time (workflow rules)
```

Comment:
I'm not sure if we should check for error here and on line 89 as well, but happy to go without for now.